src/global_styling/variables/_colors.ts

```diff
@@ -55,30 +55,38 @@ const textVariants: _EuiThemeTextColors = {
   },
 
   textSubdued: {
-    LIGHT: computed(([mediumShade]) => makeHighContrastColor(mediumShade), [
-      'colors.mediumShade',
-    ]),
-    DARK: computed(([mediumShade]) => makeHighContrastColor(mediumShade), [
-      'colors.mediumShade',
-    ]),
+    LIGHT: computed(({ colors }) =>
+      makeHighContrastColor(colors.mediumShade)(colors.pageBackground)
+    ),
+    // Or like this?
+    // LIGHT: computed(({ colors }) =>
+    //   makeHighContrastColor(colors.mediumShade, colors.pageBackground)
+    // ),
+    // Or like this?
+    // LIGHT: computed(makeHighContrastColor('colors.mediumShade')),
```

Comment:
Are you saying these are all currently possible or asking which is preferred?

Reply:
_Preferably_, it would be this last one, where the background color is optional but should just accept a simple string.

Reply:
I also never really understood the double function syntax `function()()`

Reply:
Not currently possible, but _could_ be possible. And I think I agree with your preference. We'll eventually get it typed so that we get autocomplete support for the string values.